foundation/foundation-mda/src/main/java/com/boozallen/aiops/mda/metamodel/element/StepElement.java

```diff
@@ -174,6 +174,15 @@ public List<ConfigurationItem> getConfiguration() {
 	public List<String> getFileStores() {
 		return fileStores;
 	}
+
+	/**
+	 * Sets the file stores for this step by name
+	 *
+	 * @param fileStores - a list of file store names
+	 */
+	public void setFileStores(List<String> fileStores) {
+		this.fileStores = fileStores;
+	}
```

Comment:
I: An earlier scenario iteration attached file stores to the pipeline step as well; however, after reviewing the docs it doesn't seem that file stores MUST be attached to the step to function so I removed that aspect of the scenario. I left this setter on the Step Element as it seems like a useful utility. I can remove if this doesn't make sense.